File: src/main/java/no/madsopheim/viatrumf/scraper/bff/LivenessProbe.java
package no.madsopheim.viatrumf.scraper.bff;

import org.eclipse.microprofile.health.HealthCheck;
import org.eclipse.microprofile.health.HealthCheckResponse;
import org.eclipse.microprofile.health.Liveness;

@Liveness
public class LivenessProbe implements HealthCheck {
    @Override
    public HealthCheckResponse call() {
        return HealthCheckResponse.up("Viatrumf-scraper-bff liveness");
    }
}

File: src/main/java/no/madsopheim/viatrumf/scraper/bff/infrastruktur/LivenessProbe.kt
package no.madsopheim.viatrumf.scraper.bff.infrastruktur

import org.eclipse.microprofile.health.HealthCheck
import org.eclipse.microprofile.health.HealthCheckResponse
import org.eclipse.microprofile.health.Liveness

@Liveness
class LivenessProbe : HealthCheck {
    override fun call(): HealthCheckResponse = HealthCheckResponse.up("Viatrumf-scraper-bff liveness")
}
